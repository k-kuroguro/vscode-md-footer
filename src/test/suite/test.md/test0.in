# test0

[Link1](http://www.example.com/link1)
[Link2](http://www.example.com/link2)
[Link1-2](http://www.example.com/link1)
