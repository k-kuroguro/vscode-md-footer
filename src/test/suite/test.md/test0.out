# test0

[Link1][1]
[Link2][2]
[Link1-2][1]


[1]:http://www.example.com/link1
[2]:http://www.example.com/link2
